lib/SIL/IR/SILDeclRef.cpp

```diff
@@ -256,6 +256,12 @@ SILLinkage SILDeclRef::getLinkage(ForDefinition_t forDefinition) const {
   if (isClangImported())
     return SILLinkage::Shared;
 
+  // Enum case constructors have shared linkage if the enum is internal.
```

Comment:
This part should not be necessary, just setting the limit for all enum elements is sufficient.

Reply:
Sorry, I thought we needed both. I have removed it now!